geonode_mapstore_client/client/static/mapstore/configs/localConfig.json

```diff
@@ -3097,15 +3171,71 @@
                 "cfg": {
                     "leftMenuItems": [
                         {
-                            "type": "link",
-                            "href": "{(state('gnResourceData') || {}).detail_url + '/metadata'}",
-                            "labelId": "gnviewer.editMetadata",
-                            "perms": [{ "type": "resource", "value": "change_resourcebase" }]
+                            "labelId": "gnviewer.save",
+                            "type": "dropdown",
+                            "items": [
+                                {
+                                    "type": "plugin",
+                                    "name": "Save",
+                                    "variant": "default",
+                                    "style": {
+                                        "backgroundColor": "var(--gn-main-bg, #ffffff)"
+                                    }
```

Comment:
please remove style and variant from the configuration this should be provided directly by the ActionNavbar + we should avoid inline style and use a custom className instead